odoo/addons/test_assetsbundle/tests/test_js_transpiler.py

```diff
@@ -247,15 +247,16 @@ def test_06_import(self):
 
 import Line9  from "test.Dialog";
 import  { Line10, Notification }  from 'test.Dialog2';
+import Line11  from "test/Dialog";
 
-import * as Line11 from "test.Dialog";
+import * as Line12 from "test.Dialog";
```

Comment:
... maybe don't use a sequence if that requires this sort of completely useless edits? Just call the new line something else, or add it at the end.